configuration/exodus-pipeline.yaml

```diff
@@ -260,9 +283,26 @@ Resources:
                   - Name: BuildArtifact
                 RunOrder: 2
           - !Ref AWS::NoValue
+        - !If
+          - IsProd
+          - Name: Upload
+            Actions:
+              - Name: Upload
+                ActionTypeId:
+                  Category: Deploy
+                  Owner: AWS
+                  Provider: S3
+                  Version: 1
+                Region: !Ref region
+                Configuration:
+                  BucketName: !Sub ${project}-pipeline-artifacts
+                  ObjectKey: !Sub build-${env}
+                  Extract: false
+                InputArtifacts:
+                  - Name: BuildArtifact
+          - !Ref AWS::NoValue
     DependsOn:
       - CodePipelineRole
-      - CodePipelineBucket
```

Comment:
Did you mean to remove this? Should definitely remain.

Reply:
Yes. I filed https://projects.engineering.redhat.com/browse/RHELDST-1363 to discuss this issue.